addons/mail/static/src/attachments/attachment_viewer.js

```diff
@@ -231,4 +238,19 @@ export class AttachmentViewer extends Component {
                 </html>`);
         printWindow.document.close();
     }
+
+    /**
+     *
+     * @param {('transform'|'navigation')} type
+     * @param {Event} ev
+     * @returns Boolean
+     */
+    handleEvent(type, ev) {
+        switch (type) {
+            case "transform":
+                return this.state.attachment.isImage;
+            case "navigation":
+                return true;
+        }
+    }
```

Comment:
Because this function is useless when `"navigation"` is passed, all uses of this function with `"transform"` (the rest) is just equivalent to `this.state.attachment.isImage`. I would just replace all the conditions by that.